src/main/java/mods/eln/Eln.java

```diff
@@ -5515,6 +5546,13 @@ void recipeHeatFurnace() {
 				Character.valueOf('i'), findItemStack("Copper Thermal Cable"),
 				Character.valueOf('I'), findItemStack("Combustion Chamber"));
 
+		addRecipe(findItemStack("Fuel Heat Furnace"),
```

Comment:
With liquid fuel, the main cost is the fuel itself and I'd probably default to a realistic-ish but reasonably simple recipe. That said...

Some sort of regulation for amount-of-fuel-accepted seems like it'd make sense, but these don't need power, so a motor wouldn't fit. If they're sufficiently powerful (How many 200V turbines can one of these power?), then having it use a bit of electricity could make sense -- though we're generally treating signals-level power as free, and given its nature there could be a small peltier in there anyway.

Er, balancing, right. This depends on how powerful they really are, but assuming this can replace at least two heat furnaces, I'd suggest adding a cauldron in the middle and swapping out one of the side iron ingots for a motor. You may or may not add code to make it actually need power; that's not such a big deal.

If the smallest fuel burner *doesn't* have the capacity to replace at least one fully-upgraded heat furnace -- then it should. It's generally easier to get charcoal than fuel, and it's definitely easier to handle, so players should get some benefit from going the liquid fuel path.